tools/browsers/BUILD.bazel

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can we have a readme that briefly captures what you wrote in `browser_repositories.bzl`? (can be just copied)

Also, are there plans to move this change upstream? link to an issue if present?